device-types/Dell/PowerEdge_R440.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Dell
+model: PowerEdge R440
+slug: dell_poweredge_r440
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: Rear Serial
+    type: de-9
+  - name: USB
+    type: usb-a
+  - name: Front VGA
+    type: other
+  - name: Rear VGA
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front Dedicated iDRAC direct USB
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
+  - name: Rear Dedicated iDRAC direct USB
+    type: usb-a
```

Comment:
We are no longer modelling USB ports unless they function solely as a console server port.

The iDRAC port should be modelled as a interface as that is how it functions